### ISSUE TITLE
Unknown symbols appear in Pillow import

### ISSUE THREAD

**Issue by tsulala** [microsoft/pylance-release#1465]
**Describe the bug**
It should be a correct import, but when I import the Pillow module, it appears with an unknown symbol.

**To Reproduce**
```python
from PIL import Image  # ok
from PIL import ImageCms, ImageTk, ImageTransform  # no, there are others...
```

**Expected behavior**
I was hoping that Pillow's type hints would be supported.

**Screenshots or Code**
![image](https://user-images.githubusercontent.com/39614590/122505341-2061ef80-d037-11eb-8ae1-fae28772a142.png)

**VS Code extension or command-line**
Pylance version: `2021.6.2`

**Additional context**
I think the reason for this issue is that the Pillow stubs in typeshed are not included in Pyright and refer to the old Pillow stubs in  [microsoft/python-type-stubs](https://github.com/microsoft/python-type-stubs).

The latest stub for Pillow has been recently added to `typeshed`. (python/typeshed [#5594](https://github.com/python/typeshed/pull/5594))

**Comment by jakebailey** [comment]
FWIW, pyright never shipped any stubs from python-type-stubs; those are there and are shipped with Pylance. The ones that were added to typeshed have more of the modules present, but I'd still like to compare (as the stubs I wrote weren't really used to create what was in typeshed, unfortunately).

**Comment by jakebailey** [comment]
The next release of pyright and pylance will include the stubs from typeshed, which should include that `ImageCms` module.

I've sent https://github.com/python/typeshed/pull/5692 and https://github.com/python/typeshed/pull/5706 to upstream the hand-written types I've been maintaining in https://github.com/microsoft/python-type-stubs for Pylance, and they should make it into our next release.

**Comment by tsulala** [comment]
Thanks for the fix.

**Comment by jakebailey** [comment]
This would have been fixed in 2021.7.0, but thanks to my brainfart, I forgot to delete the old stubs. It'll be fixed in a 2021.7.1 hotfix momentarily.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.7.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202171-6-july-2021
